04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -20,55 +20,62 @@
 
 
 ## Opening Framing (15m)
-We now have a way to persist data in a database. We've also learned about OOP in Ruby how we can represent things in real life programmatically as objects. Which is AWESOME! But really databases just seems like data in this kind of cryptic place on our local computer.  We have to make super long SQL statements to do CRUD. It'd be really nice if we had some kind of way to interface between the database and our servers/applications in order to streamline this process. Enter ORM's.
+Think about what we have learned so far in this unit. We now have a way to persist data in a database. We've also learned about how OOP allows us to programmatically represent real things as objects in ruby. Which is AWESOME! But really databases just seems like data in this kind of cryptic place on our local computer.  We have to make super long SQL statements to do CRUD. It'd be really nice if we had some kind of way to interface between the database and our servers/applications in order to streamline this process. Enter ORM's.
```

Comment:
can you think of another example here to really explain the word "interface"? i.e. you can drive a car without knowing how an engine works.